src/components/table/index.tsx

```diff
@@ -290,12 +295,12 @@ export default function EnhancedTable<T extends TableRowType>({
   );
 
   const onReorderEndHandler = useCallback(
-    ({ oldIndex, newIndex }) => {
+    (indexes: IndexObject) => {
```

Comment:
optionally 
```
 ({ oldIndex, newIndex }: IndexObject) => {
```
and skip the other changes below

Reply:
That was my first intention but I was getting a linter error

Reply:
What was the error? We use this pattern in the codebase today. Just curious anyways.

Reply:
Maybe I was doing something wrong there, not seeing it again. Thanks!

Reply:
Nice!